tests/js/spec/components/IssueList.spec.tsx

```diff
@@ -0,0 +1,167 @@
+import {fireEvent, screen, waitFor} from '@testing-library/react';
```

Comment:
```suggestion
```

I think we should add some info when importing this 😄

Reply:
Yeah, I keep making that mistake :D but now I know, so I'll just go forward with this :)